content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a certificate in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
```

Comment:
The signing is done with a private key stored in AKV. It would be good to keep consistent with the terminology.

Reply:
good catch, fixed